contrib/pipelines/secadvisor/README.md

```diff
@@ -2,33 +2,223 @@
-sudo $(which skydive)) allineone -c skydive.yml
+// adjust settings in skydive.yml
+sudo $(which skydive) allineone -c skydive.yml
+```
+
+### Multi-node Deployment
+Alternatively, run a skydive agent on each of your hosts and a skydive analyzer on one host.
+
+On each machine:
+
+```
+sudo skydive agent -c skydive.yml
+```
+
+Be sure to set the field "analyzers" in the skydive.yml to point to the analyzer.
+
+On one machine:
+
+```
+sudo skydive analyzer -c skydive.yml
+```
+
+
+## Create Security Advisor (Pipeline) Configuration File
+
+Start from the secadvisor.yml.default in the secadvisor directory.
+
+
+```
+cp secadvisor.yml.default secadvisor.yml
+```
+
+The secadvisor.yml is used as a parameter when running secadvisor from the command line.
+
+The default secadvisor.yml has the following fields:
+
 ```
+host_id: ""
+analyzers:
+  127.0.0.1:8082
+pipeline:
+  analyzer:
+    subscriber_url: ws://127.0.0.1:8082/ws/subscriber/flow
+    subscriber_username:
+    subscriber_password:
+  classify:
+    # cluster_net_masks:
+      # - 10.0.0.0/8
+      # - 172.16.0.0/12
+      # - 192.168.0.0/16
+  filter:
+    # excluded_tags:
+      # - internal
+      # - other
+      # - ingress
+      # - egress
+  transform:
+    sa:
+      # exclude_started_flows: true
+  store:
+    type: s3
+    s3:
+      # -- client parames --
+      endpoint: http://127.0.0.1:9000
+      region: local
+      bucket: bucket
+      access_key: user
+      secret_key: password
+      # api_key: key
+      # iam_endpoint: https://iam.cloud.ibm.com/identity/token
+      object_prefix: logs
+      # -- bulk store params --
+      max_flows_per_object: 6000
+      max_seconds_per_object: 60
+      max_seconds_per_stream: 86400
+      max_flow_array_size: 100000
+```
+
+Entries that begin with '#' are comments and are ignored. Be sure to include/exclude the appropriate configuration parameters.
+
+Each instance of secadvisor should use a distinct host_id or should use "" (the empty host id).
+If a host_id (other than "") is in use by one instance of the security advisor (pipeline), it may not be used by another instance of the security advisor (pipeline).
+
+Set "endpoint" to the object store endpoint URL, e.g. https://s3.us.cloud-object-storage.appdomain.cloud.
+Set the credentials used for authenticating to the object store service.
+You can either use AWS-style HMAC keys by setting "access_key" and "secret_key", or use IBM IAM OAuth by setting "api_key".
+When using IBM IAM Oauth (when setting "api_key"), the default IBM IAM authentication endpoint will be used: https://iam.ng.bluemix.net/oidc/token, if none is specified.
+You can customize this endpoint by setting a different "iam_endpoint" to an alternative URL string.
+
+Set "bucket" to the name of the destination bucket where flows will be dumped.
+If your object store service requires a region specification, set "region" appropriately. This setting is not necessary when using IBM COS.
```

Comment:
in the IBM cloud section you needn't mention region, if you later add a section about working against Amazon S3 you may consider explaining about region.